Tests/Transformer/ModelToElasticaAutoTransformerTest.php

```diff
@@ -473,6 +483,54 @@ public function testThatNestedObjectsDontNeedAnIdentifierField()
         ), $data['subWithoutIdentifier']);
     }
 
+    public function testNestedTransformHandlesSingleObjects()
+    {
+        $transformer = $this->getTransformer();
+        $document    = $transformer->transform(new POPO(), array(
+            'upper' => array(
+                'type' => 'nested',
+                'properties' => array('name' => null)
```

Comment:
@stof Have a look at the previous build:

https://travis-ci.org/FriendsOfSymfony/FOSElasticaBundle/builds/116368082

All tests except PHP 5.3 passed. It says:

> FOS\ElasticaBundle\Tests\Transformer\ModelToElasticaAutoTransformer\ModelToElasticaAutoTransformerTest::testNestedTransformHandlesSingleObjects
Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: Neither the property "~" nor one of the methods "get~()", "~()", "is~()", "has~()", "__get()" exist and have public access in class "stdClass".

This was the original code:

```php
    public function testNestedTransformHandlesSingleObjects()
    {
        $transformer = $this->getTransformer();
        $document    = $transformer->transform(new POPO(), array(
            'upper' => array(
                'type' => 'nested',
                'properties' => array('name' => '~')
            )
        ));
        $data = $document->getData();
        $this->assertEquals('a random name', $data['upper']['name']);
    }
```

I'm trying to understand why it failed on PHP 5.3, and passed on PHP 5.4/5.5/5.6/7.